File: packages/tasks-gen/snippets-fixtures/text-to-speech/python/requests/0.fal-ai.py
import os
import requests

API_URL = "https://router.huggingface.co/fal-ai/<fal-ai alias for nari-labs/Dia-1.6B>"
headers = {
    "Authorization": f"Bearer {os.environ['HF_TOKEN']}",
}

def query(payload):
    response = requests.post(API_URL, headers=headers, json=payload)
    return response.json()

audio, sampling_rate = query({
    "text": "The answer to the universe is 42",
})
# You can access the audio with IPython.display for example
from IPython.display import Audio
Audio(audio, rate=sampling_rate)

File: packages/tasks-gen/snippets-fixtures/text-to-speech/js/fetch/0.fal-ai.js
async function query(data) {
	const response = await fetch(
		"https://fal.run/<fal-ai alias for nari-labs/Dia-1.6B>",
		{
			headers: {
				Authorization: "Key api_token",
				"Content-Type": "application/json",
			},
			method: "POST",
			body: JSON.stringify(data),
		}
	);
    const result = await response.json();
    return result;
}

query({ text: "The answer to the universe is 42" }).then((response) => {
    console.log(JSON.stringify(response));
});